docs/telemetry.md

```diff
@@ -1,53 +1,386 @@
-# Telemetry on Percona Distribution for PostgreSQL
+# Telemetry and data collection
 
-Percona telemetry fills in the gaps in our understanding of how you use Percona Distribution for PostgreSQL to improve our products. Participation in this anonymous program is optional. You can opt-out if you prefer to not share this information.
+Percona collects usage data to improve its software. The telemetry feature helps us identify popular features, detect problems, and plan future improvements. 
```

Comment:
Can we maybe mention here already that we do not pull any sensitive data, data is basically anonymized

Reply:
Added